src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -7,6 +7,8 @@
     <RootNamespace>Microsoft.MIDebugPackage</RootNamespace>
     <AssemblyName>Microsoft.MIDebugPackage</AssemblyName>
     <OutputPath>$(MIDefaultOutputPath)</OutputPath>
+
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
```

Comment:
Rather than having to hack up this project, should we just move our XPlat packaging to something more sensible?

If that is too hard: instead of doing this, should we just exclude building resources in XPlat configurations?

Reply:
I believe I will need a new project for `dotnet publish` when we make the VS Code projects build for .NET 5. I'll exclude building resources for now and then do the splitting in another PR.